src/plugins/health.ts

```diff
@@ -201,7 +201,7 @@ export class Health {
    *
    * This function is only available on Android.
    *
-   * @return successCallback: {type: function()}, called if the function was called
+   * @return  {Promise<any>} {type: function()}, called if the function was called
```

Comment:
All of this should be removed: 
```
 {type: function()}, called if the function was called
     * errorCallback: {type: function(err)}, called if something went wrong		     * errorCallback: {type: function(err)}, called if something went wrong
```

All what it needs is `@return {Promise<any>}` 

You can add additional description such as: 
```
@returns {Promise<any>} returns a promise that resolves when (something happens) and rejects if (something else happens).
```